_application_development/readme.md

```diff
@@ -0,0 +1,8 @@
+---
+title: README
+position: 2
+---
+
+Your project's `README.md` is your project's first and last line of defense against unnecessary overhead and complexity for future maintainers or contributors to the application.
```

Comment:
Do we want to have a strategy around making a readme that's designed for dev/contribution vs installing/running?

This is especially important for microservices that people will want to integrate into other projects

Reply:
@briaguya hmmmmm I've been thinking about this comment since I first read it a few days ago. I'm inclined to say that microservice usage documentation might be more appropriate in either a repo's Wiki or as a stand-alone microsite, or perhaps even hosted on a 3rd-party tool like APIary. The thinking there is that robust documentation might be more appropriately published with some file structure and content organization demands beyond a single flat file. Or perhaps the maintainers can determine whether their usage is simple enough to include in `README` or if they think it demands a larger effort that's linked to from the `README`. Maybe we can revisit this once we have our first microservice stood up and can address this question for that particular project?